encrypted-media-respec.html

```diff
@@ -4540,6 +4570,11 @@
       </section>
     </section>
 
+    <section id="acknowledgements">
+      <h2>Acknowledgments</h2>
+      The editors would like to thank <a def-id="contributors"></a> for their contributions to this specification.  Thank you also to the many others who contributed to the specification, including through their participation on the mail list and in the issues. 
```

Comment:
nit: We use "mail**ing** list" in the SotD. Either way is fine, but I suggest being consistent.